Dockerfile

```diff
@@ -55,4 +50,4 @@ ENV PORT=8443
 # expose port 8443 by default
 EXPOSE 8443
 
-CMD ["pm2-runtime", "start", "ecosystem.config.js"]
+ENTRYPOINT [ "npm", "start" ]
```

Comment:
Would it be good idea to use `pnpm` here so it doesn't require using `--` to pass arguments down to `nuxt` CLI?

Reply:
Would that require us to explicitly install pnpm after the `FROM node:alpine as app
` line? Alternatively I could add the '--' to the end of the entrypoint array.

Reply:
Ahh I suppose so. It would be nice to avoid that so we don't have to add that extra installation time. Another alternative is that we could call to the nuxt CLI directly:

```
ENTRYPOINT [ "node", "./node_modules/nuxt/bin/nuxt.js", "start" ]
```

It's not that important though, whatever you think is easiest and most understandable.

Reply:
```suggestion
ENTRYPOINT [ "npm", "start", "--" ]
```